src/cloud-api-adaptor/test/e2e/common.go

```diff
@@ -30,7 +30,8 @@ import (
 const WAIT_DEPLOYMENT_AVAILABLE_TIMEOUT = time.Second * 180
 const DEFAULT_AUTH_SECRET = "auth-json-secret-default"
 
-var testInitdata string = `algorithm = "sha384"
+var certContent string
+var testInitdatahttps string = `algorithm = "sha384"
```

Comment:
nit: If we are renaming this, it should have the HTTPS in capitals to match convention

Reply:
Also if renamed - you need to update `azure_test.go`